src/Sulu/Bundle/MediaBundle/Collection/Manager/DefaultCollectionManager.php

```diff
@@ -19,11 +23,8 @@
 use Sulu\Bundle\MediaBundle\Media\Exception\CollectionNotFoundException;
 use Sulu\Bundle\MediaBundle\Media\Exception\CollectionTypeNotFoundException;
 use Sulu\Bundle\MediaBundle\Media\FormatManager\FormatManagerInterface;
-use Sulu\Component\Rest\ListBuilder\Doctrine\FieldDescriptor\DoctrineJoinDescriptor;
 use Sulu\Component\Rest\ListBuilder\Doctrine\FieldDescriptor\DoctrineFieldDescriptor;
-use Doctrine\Common\Persistence\ObjectManager;
-use Sulu\Bundle\MediaBundle\Entity\Collection as CollectionEntity;
-use Sulu\Bundle\MediaBundle\Api\Collection;
+use Sulu\Component\Rest\ListBuilder\Doctrine\FieldDescriptor\DoctrineJoinDescriptor;
 use Sulu\Component\Security\Authentication\UserRepositoryInterface;
 
 class DefaultCollectionManager implements CollectionManagerInterface
```

Comment:
should we really change such stuff in a PR which is not related to this? we shouldnt refactor in a PR which is such big as this ...